<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindowClass</class>
 <widget class="QMainWindow" name="SettingsWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>636</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>SettingsWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pB_SettingsBack">
        <property name="maximumSize">
         <size>
          <width>145</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidgetSettings">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabMoreSettings">
       <attribute name="title">
        <string>Seite</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>288</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>12</pointsize>
                <underline>true</underline>
               </font>
              </property>
              <property name="text">
               <string>More Settings</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_LogFilePath">
              <property name="text">
               <string>Logfile Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_logFilePath">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pB_selectPath">
              <property name="text">
               <string>Select Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_mySQLusername">
              <property name="text">
               <string>MySQL Username</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mySQLusername"/>
            </item>
            <item>
             <widget class="QLabel" name="lb_mySQLDatabase">
              <property name="text">
               <string>MySQL Database</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mySQLDatabase">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_mySQLHostename">
              <property name="text">
               <string>MySQL Hostename</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mySQLHostename">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_mySQLPort">
              <property name="text">
               <string>MySQL Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mySQLPort">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_mySQLPassword">
              <property name="text">
               <string>MySQL Password</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="le_mySQLPassword">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::EchoMode::Password</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_Style">
              <property name="text">
               <string>Style - Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_Style">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Light</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Light Dark</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark Flat</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New02</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New03</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>New04</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lb_language">
              <property name="text">
               <string>Language - Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cB_language">
              <property name="maximumSize">
               <size>
                <width>400</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>English</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>German</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pB_SettingsSave">
              <property name="text">
               <string>Save</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_16">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>344</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../InventoryManagementSystem.qrc"/>
 </resources>
 <connections/>
</ui>
